Bottom: 478b5a9511d73300038688d295275adc008791fb
Top:    c1f6d017ec3215a105705280d9d4e90b49cdb2fc
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-17 01:25:08 -0300

Thread safety documentation.

for ChangeLog

	* manual/math.texi: Document thread safety properties.


---

diff --git a/manual/math.texi b/manual/math.texi
index 193d415..fd5d0c8 100644
--- a/manual/math.texi
+++ b/manual/math.texi
@@ -560,6 +560,7 @@ negative), @code{logb} returns @math{@infinity{}}.  If @var{x} is zero,
 @comment math.h
 @comment ISO
 @deftypefunx int ilogbl (long double @var{x})
+@safety{@mtsafe{}@assafe{}@acsafe{}}
 These functions are equivalent to the corresponding @code{logb}
 functions except that they return signed integer values.
 @end deftypefun
